packages/react-dom/src/__tests__/ReactDOMRoot-test.js

```diff
@@ -119,6 +119,21 @@ describe('ReactDOMRoot', () => {
     expect(callback).not.toHaveBeenCalled();
   });
 
+  it('double invokes useLayoutEffect in __DEV__', () => {
```

Comment:
Should we use `// @gate __DEV__` to gate this test? It is used elsewhere in the codebase to gate tests only for DEV mode.